python/msbuild/ice.proj

```diff
@@ -46,6 +46,15 @@
         <PythonGenerated Include="$(MSBuildThisFileDirectory)\..\python\**\*_ice.py"/>
     </ItemGroup>
 
+    <ItemGroup>
+        <ClientSliceCompile Include="$(MSBuildThisFileDirectory)..\test\**\*.ice" Exclude="**\Server*.ice">
```

Comment:
The exclude is become some tests define client-private or server-private slice files in Client*.ice or Server*.ice files?